From be3ce3fbf337246e84cf8704891beafea393f5e8 Mon Sep 17 00:00:00 2001
From: Andrew Kelley <andrew@ziglang.org>
Date: Fri, 5 Sep 2025 19:45:18 -0700
Subject: [PATCH 095/133] langref: update for new packed union rules
X-Developer-Signature: v=1; a=openpgp-sha256; l=676; i=xtex@astrafall.org;
 h=from:subject; bh=zikQ0HTtxqNiZAgz0xP2GvtFGfdGoqqJH3dojzHm6cU=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn78jiPzS7Q61OTCDg56/X1B4kXbPXsi7T6eo77/gqo5
 K5ZP9OsYyILgxgXg6WYIkuRYYM3q046v+iyclmYOaxMIEOkRRoYGBgYWBj4chPzSo10jPRMtQ31
 DA11jHSMGLg4BWCq284xMjQqu4TIfj92XCVO8oT6VAPx/W226n57N83eKHNz03P2chtGhpdnjbL
 UHqoqxtUr27z7MslY0tBubeXWPTZPKk0TCxdWMQIA
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit 8c631ebfeeed9b2338b3f4a280f6679c527f487b)
---
 doc/langref.html.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/doc/langref.html.in b/doc/langref.html.in
index 9c54fba920c7..bdfcd7f0252c 100644
--- a/doc/langref.html.in
+++ b/doc/langref.html.in
@@ -2496,6 +2496,7 @@ or
       {#header_open|packed union#}
       <p>A {#syntax#}packed union{#endsyntax#} has well-defined in-memory layout and is eligible
           to be in a {#link|packed struct#}.</p>
+      <p>All fields in a packed union must have the same {#link|@bitSizeOf#}.</p>
       {#header_close#}
 
       {#header_open|Anonymous Union Literals#}
-- 
2.52.0

